src/Moryx.Products.Management/Facades/ProductManagementFacade.cs

```diff
@@ -256,5 +287,25 @@ private static void ValidateRecipe(IProductRecipe recipe)
             recipe.Origin = this;
             return recipe;
         }
+
+        public bool DeleteProduct(long id)
+        {
+            throw new NotImplementedException();
+        }
+
+        private class IsConfiguredFilter
+        {
```

Comment:
Relict of previous version?

Reply:
No I actually forgot to implement the method 🤦‍♂️